docs/ValidationCriteriaMaster.md

```diff
@@ -0,0 +1,304 @@
+# Validation Criteria Master Document
+This document contains a comprehensive list of validation criteria referenced by all Test Instructions.
+
+Its objective is to guarantee consistency of criteria, allowing the authors to refer to the same table format.
+
+> **ATTENTION**: for some rules a test-specific VALUE might be needed. In this case the tables below just capture an example between []
+
+
+
+## Utilities
+ [Table converter](https://tableconvert.com/) (e.g., Excel to MD, MD to MD)
+
+<details><summary>Alignment Table template</summary>
+
+| Entity                        | PredefinedType | Name | Start Point X | Start Point Y | Start Direction | Start Radius Of Curvature | End Radius Of Curvature | Segment Length |
+|-------------------------------|----------------|------|---------------|---------------|-----------------|---------------------------|-------------------------|----------------|
+| IfcAlignmentHorizontalSegment | LINE           | H1   | 452413.9199   | 4539456.401   | 0.214271681     | 0                         | 0                       | 234.719412     |
+| IfcAlignmentHorizontalSegment | CLOTHOID       | H2   | 452634.415    | 4539536.869   | 0.214271681     | 0                         | -1000                   | 40             |
+| IfcAlignmentHorizontalSegment | CIRCULARARC    | H3   | 452671.898    | 4539550.832   | 0.236493911     | -1000                     | -1000                   | 193.464471     |
+
+</details>
+
+<details><summary>Entities Table template</summary>
+
+> Agree on template table
+
+> **OPTION 1**: strictly just the attributes of the entity
+
```

Comment:
- Entities template table: I think that it is always nice to have a notes column (option 2)

Reply:
Sold!